impl/src/test/java/org/ehcache/docs/GettingStarted.java

```diff
@@ -37,7 +37,7 @@
 import org.ehcache.config.serializer.DefaultSerializerConfiguration;
 import org.ehcache.config.units.EntryUnit;
 import org.ehcache.config.units.MemoryUnit;
-import org.ehcache.config.writebehind.WriteBehindConfigurationBuilder;
+import org.ehcache.config.loaderwriter.writebehind.WriteBehindConfigurationBuilder;
```

Comment:
I would have expected much more changes in this class. The builder changes most likely impact the getting started as we want to promote the easy path first.